fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/PermissiveAvailability.java

```diff
@@ -53,7 +51,11 @@ public PermissiveAvailability(
 
     @Override
     public Map<Column, Set<Interval>> getAllAvailableIntervals() {
-        Map<String, Set<Interval>> allAvailableIntervals = metadataService.getAvailableIntervalsByTable(tableName);
+        Map<String, Set<Interval>> allAvailableIntervals = metadataService.getAvailableIntervalsByTable(
+                dataSourceNames.stream()
+                        .findFirst()
+                        .orElseThrow(NullPointerException::new)
```

Comment:
This is not necessary since the constructor only allows a single name to be passed in, also throwing a null pointer exception with out any additional information is not recommended.